components/feature/addons/src/main/res/layout/mozac_feature_addons_item.xml

```diff
@@ -28,8 +28,8 @@
                 android:layout_width="32dp"
                 android:layout_height="32dp"
                 android:layout_gravity="center"
-                android:src="@drawable/mozac_ic_extensions"
                 android:importantForAccessibility="no"
+                android:src="@android:color/transparent"
```

Comment:
`@null` works better here I think, not sure if transparent is the best practice for this fwiw :)

Reply:
Great observation 👍  `@null` is better for performance, but in this case we want to have a `drawable` to transitioning from it to the new icon and if we use `@null` `image.getDrawable() `will [return null](https://stackoverflow.com/a/9311334/5533820) and cause the app to crash [here](https://github.com/mozilla-mobile/android-components/blob/270857e1e0ccf1dcec6bd29611f2ce0763b8aa27/components/feature/addons/src/main/java/mozilla/components/feature/addons/ui/AddonsManagerAdapter.kt#L292)